package.json

```diff
@@ -34,6 +34,9 @@
     "jwk-to-pem": "^1.2.6",
     "uuid": "^3.2.1"
   },
+  "peerDependencies":{
+    "aws-sdk": "^2.0.0"
+  },
   "devDependencies": {
     "aws-sdk": "2.457.0",
```

Comment:
Do we still need `aws-sdk` as peer dependency?

Reply:
Sorry, I was asleep when writing the comment.

What I meant is: do we need it in `devDependencies` or should we actually drop it from there because we want it only as peer dependency?

Reply:
The tests will fail if we remove it from devDependencies

Reply:
Then, I guess, it makes sense to keep it in devDependencies. We are still achieving what was originally stated in https://github.com/Cimpress-MCP/microservice-utilities.js/issues/33.